src/Admin/SiteHealth.php

```diff
@@ -171,6 +182,78 @@ public function persistent_object_cache() {
 		];
 	}
 
+	/**
+	 * Get the test result data for whether there is page cache or not.
+	 *
+	 * @return array
+	 */
+	public function page_cache() {
+
+		$is_using_page_cache = $this->is_site_has_page_cache();
+
+		return [
+			'badge'       => [
+				'label' => $this->get_badge_label(),
+				'color' => $is_using_page_cache ? 'green' : 'orange',
+			],
+			'description' => esc_html__( 'The AMP plugin performs at its best when page object cache is enabled.', 'amp' ),
+			'test'        => 'amp_page_cache',
+			'status'      => $is_using_page_cache ? 'good' : 'recommended',
+			'label'       => $is_using_page_cache
+				? esc_html__( 'Page caching is enabled', 'amp' )
+				: esc_html__( 'Page caching is not enabled', 'amp' ),
+		];
+	}
+
+	/**
+	 * Check if site has page cache enable or not.
+	 *
+	 * @return bool True if site has page cache. Otherwise False.
+	 */
+	private function is_site_has_page_cache() {
+
+		$request_url    = home_url( '?amp_page_cache=1' );
+		$has_page_cache = false;
+		$response_list  = [];
+		$request_args   = [];
+
+		for ( $i = 1; $i <= 3; $i++ ) {
+			$response = wp_remote_get( $request_url, $request_args );
```

Comment:
2nd argument for `wp_remote_get()` is an empty array by default.

```suggestion
		for ( $i = 1; $i <= 3; $i++ ) {
			$response = wp_remote_get( $request_url );
```